deployer/machine.go

```diff
@@ -14,6 +14,7 @@ func StateMachine() (*machine.StateMachine, error) {
     "States": {
       "Validate": {
         "Type": "TaskFn",
+        "Resource": "arn:aws:lambda:{{aws_region}}:{{aws_account}}:function:{{resource_name}}",
```

Comment:
Since the client is renaming this `{{resource_name}}` it should know what the resource is (a lambda) so `lambda_name` I think makes more sense. Then later it would be `{{activity_name}}` or what ever. Also since the client is the one that is string replacing these this is more a convention than a standard.